<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1336</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Panager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resorsy.qrc">
    <normaloff>:/new/prefix1/ico.png</normaloff>:/new/prefix1/ico.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
/*	color: #f56f18;*/
color: #yelllow;
	/*background-color:#ba4a00;*/
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #220d00, stop: 1 #220d00);
	background-image:url(:/new/prefix1/background.png);
	 background-repeat: repeat-x;


}


QPushButton {
/*	color:#494b7a;*/
	color:#2f304f;
    border-width: 4px;
 /*   border-image: url(:/new/prefix1/button.png) 4 4 4 4 stretch stretch;*/
/*border-image:url(:/new/prefix1/button.png) stretch stretch;*/
border-image:url(:/new/images/button2.png);

	}

QPushButton:hover{
	color:#5d609c;
	border-image: url(:/new/images/button2Nac.png)  stretch stretch;;

}
QTextEdit:focus{
	background-color:#ffffff;
}
#statusBar{
	color:white;
}

QStatusBar{
	color:white;
}





</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">
QTextEdit{
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #999999, stop: .01 #aaaaaa,stop: .1 #eeeeee, stop: 0.7 #ffffff, stop: 1 #eeeeee);
color:#333333;
}

QTextEdit:focus{
	background: #ffffff;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>850</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>850</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">#scrollArea{
	background-color: #ffffff;
}</string>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>844</width>
           <height>688</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">#scrollAreaWidgetContents{
	background-color:#ffffff;
}</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&amp;OK</string>
          </property>
          <property name="shortcut">
           <string>Alt+O</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>&amp;Exit</string>
          </property>
          <property name="icon">
           <iconset resource="resorsy.qrc">
            <normaloff>:/new/images/close.png</normaloff>:/new/images/close.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>live</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>500</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>464</width>
           <height>200</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTableWidget" name="tableWidget"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButton_plus">
          <property name="text">
           <string>+</string>
          </property>
          <property name="shortcut">
           <string>Ctrl++</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_equal">
          <property name="text">
           <string>=</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+=</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_minus">
          <property name="text">
           <string>-</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+-</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_prev">
          <property name="text">
           <string>prev</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+,</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_next">
          <property name="text">
           <string>next</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+.</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>&amp;Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_loguj">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">/*
QPushButton {
    border-width: 4px;
    border-image: url(:/new/prefix1/button.png) 4 4 4 4 stretch stretch;
 	
	font:bold;
}*/
/*
QPushButton:hover{
color: #ffffff;
}*/</string>
          </property>
          <property name="text">
           <string>&amp;Login</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1336</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#scrollArea{
	color: #ffffff;
}
QMenuBar {
     background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0 lightgray, stop:1 darkgray);
 }

 QMenuBar::item {
     spacing: 3px; /* spacing between menu bar items */
     padding: 1px 4px;
     background: transparent;
     border-radius: 4px;
 }

 QMenuBar::item:selected { /* when selected using mouse or keyboard */
     background: #a8a8a8;
 }

 QMenuBar::item:pressed {
     background: #888888;
 }
#statusBar{
	color:#ffffff;
}</string>
   </property>
   <widget class="QMenu" name="menuNew">
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>New</string>
    </property>
    <addaction name="actionRecord"/>
    <addaction name="actionPhoto"/>
   </widget>
   <widget class="QMenu" name="menuZak_adki">
    <property name="title">
     <string>Bookmarks</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <widget class="QMenu" name="menuUsage">
     <property name="title">
      <string>Usage</string>
     </property>
     <addaction name="actionProgram"/>
     <addaction name="actionC2h50h"/>
    </widget>
    <addaction name="menuUsage"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuDane_logowania">
    <property name="title">
     <string>Login data</string>
    </property>
    <addaction name="actionSave_login_and_password"/>
   </widget>
   <addaction name="menuNew"/>
   <addaction name="menuZak_adki"/>
   <addaction name="menuDane_logowania"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">QStatusBar {
	color: white;
	
}
#statusBar{
	color:#ffffff;
}</string>
   </property>
  </widget>
  <action name="actionRecord">
   <property name="icon">
    <iconset resource="resorsy.qrc">
     <normaloff>:/new/prefix1/icol.ico</normaloff>:/new/prefix1/icol.ico</iconset>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Zmodyfikuj lub &lt;span style=&quot; font-weight:600; color:#c81b52;&quot;&gt;add&lt;/span&gt;&lt;span style=&quot; font-weight:400; color:#000000;&quot;&gt;record&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionProgram">
   <property name="text">
    <string>Program</string>
   </property>
   <property name="toolTip">
    <string>Program</string>
   </property>
  </action>
  <action name="actionC2h50h">
   <property name="text">
    <string>c2h50h</string>
   </property>
  </action>
  <action name="actionSave_login_and_password">
   <property name="text">
    <string>Save login and password</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPhoto">
   <property name="text">
    <string>Photo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resorsy.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonOKHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>938</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_loguj</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showLogin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1275</x>
     <y>709</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonOKHandler()</slot>
 </slots>
</ui>
